packages/framework/attributor/src/runtimeAttributorDataStoreFactory.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IFluidDataStoreFactory,
+	IFluidDataStoreChannel,
+	IFluidDataStoreContext,
+	IFluidDataStoreRegistry,
+	FluidDataStoreRegistryEntry,
+} from "@fluidframework/runtime-definitions/internal";
+import { PerformanceEvent, createChildLogger } from "@fluidframework/telemetry-utils/internal";
+
+import { RuntimeAttributorDataStoreChannel } from "./runtimeAttributorDataStoreChannel.js";
+
+/**
+ * Factory for the runtime attributor data store channel.
+ */
+export class RuntimeAttributorFactory
+	implements IFluidDataStoreFactory, IFluidDataStoreRegistry
+{
+	public static readonly type = "@fluid-experimental/attributor";
+
+	public get type(): string {
+		return RuntimeAttributorFactory.type;
+	}
+
+	public get IFluidDataStoreRegistry(): IFluidDataStoreRegistry {
+		return this;
+	}
+
+	public get IFluidDataStoreFactory(): IFluidDataStoreFactory {
+		return this;
+	}
+
+	public async get(name: string): Promise<FluidDataStoreRegistryEntry | undefined> {
+		return undefined;
```

Comment:
Is this correct?  Should it return a registry entry for Attributor?  Or, can we remove `IFluidDataStoreRegistry` from what this factor implements?

Reply:
This was a registry for ddses or channel under it but since it does not need to create those, we can remove it.